hw3/03/src/components/Navbar.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import "@mui/material/styles";
+import { AppBar, Toolbar, Typography } from "@mui/material";
+
+function Navbar() {
```

Comment:
I don't really have any recommendations here this looks really good. I have been wanting to work with mui for a bit now so it was nice to see it being implemented. good job